viaacode/avo2-client
# Registration flow for teachers

partly closes: https://district01.atlassian.net/browse/AVO2-153

To test this PR, you'll also need 
* the proxy PR: https://github.com/viaacode/avo2-proxy/pull/56
* the typings PR: https://github.com/viaacode/avo2-typings/pull/23

Still a work in progress, but everything up until the verification email should work
You can test registration with the stamboek number in 1password (VIAA stamboek number test)

![image](https://user-images.githubusercontent.com/1710840/69548578-e08c7080-0f97-11ea-8457-fdabc0121595.png)

![image](https://user-images.githubusercontent.com/1710840/69548584-e41ff780-0f97-11ea-91d3-80571d66e76d.png)

![image](https://user-images.githubusercontent.com/1710840/69548591-e84c1500-0f97-11ea-8578-2b02c6d8cba6.png)
